test/cpp/qps/client.h

```diff
@@ -206,9 +207,6 @@ class Client {
       }
     }
 
-    grpc_stats_data core_stats;
-    grpc_stats_collect(&core_stats);
```

Comment:
why don't we collect stats for qps tests anymore?

Reply:
so it would need to be updated, but it's actually broken at the minute and causing all our oss benchmarks to fail
we'll probably want to revisit this, but with a new design

Reply:
should we leave a TODO here then?

Reply:
I think not yet - if we build something again it's unlikely it looks the same.